bin/aws-kinesis-agent.RedHat

```diff
@@ -191,6 +192,11 @@ do_install () {
   echo "$DAEMON_NAME log file will be found at: $LOG_DIR"
 }
 
+exec 200>$MUTEXFILE
+if ! flock -w 50 200; then
```

Comment:
The choice of 50 seconds here was made arbitrarily.  Probably a wiser value might be 25 seconds.